tensorflow/python/framework/python_op_gen_main.cc

```diff
@@ -78,10 +79,33 @@ Status ParseHiddenOpsCommandLine(const char* arg,
 }
 
 void PrintAllPythonOps(const std::vector<string>& hidden_ops,
+                       const string& source_file_name,
                        bool require_shapes) {
   OpList ops;
   OpRegistry::Global()->Export(false, &ops);
-  PrintPythonOps(ops, hidden_ops, require_shapes);
+  PrintPythonOps(ops, hidden_ops, source_file_name,require_shapes);
```

Comment:
Space after comma.  I think there is some program to automatically format the code, but I don't have instructions.